packages/teams-js/src/public/app.ts

```diff
@@ -699,7 +700,11 @@ export namespace app {
         processAdditionalValidOrigins(validMessageOrigins);
       }
 
-      resolve(GlobalVars.initializePromise);
+      if (GlobalVars.initializePromise !== null) {
+        resolve(GlobalVars.initializePromise);
+      } else {
+        initializeHelperLogger('GlobalVars.initializePromise is unexpectedly null');
+      }
```

Comment:
Since the compiler now realizes that `GlobalVars.initializePromise`, it (correctly) won't let you call `resolve` with something that could be `null`. Now we are logging when we would already have been in that unexpected situation.